### PR TITLE
feat: implement hw3

### PR DESCRIPTION
**Pull request by matveyprupas**
Task: https://epam.sharepoint.com/sites/LMSO/Program/31840fb43b6e-8f14-ed4a-dde5-20ff7e7160cd/Classes,%20Constructor,%20new.pdf

### CODE REVIEW

File: hw3/index.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
+class Vector {
+    constructor(arr) {
+        this.coordinates = arr;
+    }
+}
+
+Vector.prototype.add = function(vector) {
```
Comment (NikitaSpirit) [inline hw3/index.js]: @matveyprupas Why do you use `Vector.prototype` contruction to add a method?

**Review by NikitaSpirit — CHANGES_REQUESTED** [review]

File: hw3/index.js (hunk 1 above)
Comment (matveyprupas) [reply hw3/index.js]: Because I mixed up ES5 and ES6 syntax. Fixed it. Wrote everything in ES6 syntax.

**Comment by matveyprupas** [comment]
@NikitaSpirit Fixed it

**Review by NikitaSpirit — APPROVED** [review]
Looks god now. Please, ,merge it

**Pull request merged**